lib/mixpanel-node.js

```diff
@@ -19,12 +19,17 @@ var create_client = function(token, config) {
         throw new Error("The Mixpanel Client needs a Mixpanel token: `init(token)`");
     }
 
+    // Default config
     metrics.config = {
         test: false,
         debug: false,
-        verbose: false
+        verbose: false,
+        host: 'api.mixpanel.com'
     };
 
+    // Store user-provided config, overriding defaults
+    for(var i in config) { metrics.config[i] = config[i]; }
```

Comment:
Kind of? In my view, set_config() is useful to set config after initialisation, but at present the config parameter in **mixpanel.init(token, config)** is ignored. It seems to me like a reasonable user expectation for that config parameter to be used at init time. But perhaps set_config should be called during initialisation?